modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -47,7 +47,8 @@ Download the following files:
 ... The machine host and domain name in the form `hostname.domainname`. Omit this value to let {op-system} decide set it.
 ... The network interface name. Omit this value to let {op-system} decide set it.
 ... If you use static IP addresses, an empty string.
-** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role. The bootstrap machine ignition file is called `bootstrap-0`, the master ignition files are numbered 0 through 2, the worker ignition files from 0 upwards. All other parameters can stay as they are.
+** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role (`bootstrap.ign`, `master.ign`, or `worker.ign`).
```

Comment:
I'd edit this to remove the parentheses.

Please squash your commits.

Reply:
@rh-max, I have not seen that before! But GitHub had an outage yesterday, so please try rebasing and force pushing your local branch again before you make a new PR.

Reply:
@kalexand-rh Rebasing worked. Please let me know if anything else is missing.